learn/core_concepts/indexes.md

```diff
@@ -48,55 +35,91 @@ The `uid` is set at [index creation time](/reference/api/indexes.md#create-an-in
 
 ## Primary key
 
-An index is a collection of documents. All documents have a primary key, which is a mandatory field. This field is composed of a primary key attribute name and a unique value. All documents in a given index share the same primary key attribute but a different unique value.
+Every index has a primary key: a required attribute that must be present in all documents in the index. The primary key's attribute name must be known by the index. Each document in the index must have a unique value associated with the primary key. This serves to identify each document in the index, such that two documents in an index can never be completely identical. If you try adding documents to an index and even a single one is missing the primary key, none of the documents will be stored.
 
-The primary key's attribute name **must** be known by the index. You can [set a primary key for an index or let it be inferred by Meilisearch](/learn/core_concepts/primary_key.md#setting-the-primary-key).
+You can set a primary key for an index or let it be inferred by Meilisearch, read more about it in our [dedicated guide](/learn/core_concepts/primary_key.md#setting-the-primary-key).
 
 [Learn more about document primary key](/learn/core_concepts/primary_key.md#primary-key-2)
 
-## Relevancy rules
+## Index settings
+
+Index settings can be thought of as a JSON object containing different options for customizing search behavior.
+
+Meilisearch allows you to customize the following index settings:
+
+- [Ranking rules](#ranking-rules)
+- [Synonyms](#synonyms)
+- [Filterable attributes](#filterable-attributes)
+- [Sortable attributes](#sortable-attributes)
+- [Stop words](#stop-words)
+- [Displayed and searchable attributes](#displayed-and-searchable-attributes)
+- [Typo tolerance](#typo-tolerance)
 
-Each index applies its own relevancy rules. All indexes are created with the same built-in ranking rules executed in a default order. Once your first document has been added, the index will record how the attributes must be sorted. Their order of importance will be deduced from their order of appearance in the document.
+These settings are what differentiate between indexes even if they have the same documents. You can make these changes using the [settings route](/reference/api/settings.md) or the [dedicated settings child route](/reference/api/settings.md#all-settings).
```

Comment:
I'm trying to shift the emphasis from "settings differentiate/distinguish indexes" to "settings make indexes behave differently." Like, I feel that the differentiation itself is less important than the reason why you would make things different in the first place.

```suggestion
These settings can make indexes behave differently even if they have the same documents. To change them, you can use the [settings route](/reference/api/settings.md) or the [dedicated settings child route](/reference/api/settings.md#all-settings).
```